src/views/statutory/EditPositionModal.vue

```diff
@@ -80,7 +91,8 @@ export default {
         starts: null,
         ends: null,
         ends_force: null
-      }
+      },
+      bodiesWithElections: []
```

Comment:
let's rename it to just `bodies`, also we don't need to fetch it here, we can just pass it from the PositionList component as a prop